command/commands.go

```diff
@@ -136,6 +141,11 @@ const (
 	flagNameUserLockoutDisable = "user-lockout-disable"
 	// flagNameDisableRedirects is used to prevent the client from honoring a single redirect as a response to a request
 	flagNameDisableRedirects = "disable-redirects"
+	// flagNameLogFile is used to specify the path to the log file that Vault should use for logging
+	flagNameLogFile = "log-file"
```

Comment:
Similar to my comment above regarding the `VAULT_LOG_FILE` environment variable, should we denote the Agent-specific nature of this flag in the comment?

Reply:
Resolving based on comment above